style.css

```diff
@@ -1725,6 +1725,16 @@
     [data-night-mode] section.step-one > aside > .content ul li p time.date {
         color: #aaa;
     }
+
+    .comments section.stream.entry-subcomments section.entry.author > article:before,
+    .comments section.stream.entry-subcomments section.entry.link-author > article:before,
+    .comments section.stream.entry-subcomments section.entry.own > article:before {
+        left: -2px;
+    }
+
+    .comments section.stream.entry-subcomments section.entry.reply:before {
```

Comment:
Usunięcie separatorów weszło już w commicie: https://github.com/tentin-quarantino/wykop-the-best-style/commit/410cd901456e0d8861f72fce26c90963e19b3c99
Mógłbyś usunąć, aby uniknąć niepotrzebnej redundancji?